ui/lib/ldap/addon/components/accounts-checked-out.hbs

```diff
@@ -15,16 +15,15 @@
             <Body.Td data-test-checked-out-library={{Body.data.account}}>{{Body.data.library}}</Body.Td>
           {{/if}}
           <Body.Td>
-            <button
-              type="button"
+            <Hds::Button
+              @icon="queue"
+              @text="Check-in"
+              @color="secondary"
               class="text-button has-text-primary has-text-weight-semibold"
```

Comment:
It looks like this could be tertiary and we can remove all of these classes? I think Yankun made these designs using HDS components (so this is a one off)
<img width="884" alt="Screenshot 2023-10-12 at 9 46 00 AM" 
src="https://github.com/hashicorp/vault/assets/68122737/81f14b37-02b1-4476-a1f5-22cf60f963f3">

Reply:
But perhaps I have the wrong screenshot 🙃

Reply:
Tertiary looks pretty much the same to what those classes were doing so I'll go ahead and update!